### DISCUSSION TITLE
Pi 3 B+ help

### DISCUSSION THREAD

**Discussion by Bojack84** [aaronwmorris/indi-allsky#1022] [Q&A] [answered]
I have a Pi 3 B+ and every time that it runs a video creation the whole system shuts down.  I looked at the Armbian Tuning section and that hasn't helped.  Any idea on how I can help this?  It works great for the most part and I am getting some great videos from the IP Camera.

**Comment by aaronwmorris** [comment]
What resolution videos are you generating?

**Reply by Bojack84** [reply]
Do you mean this? 
![image](https://github.com/aaronwmorris/indi-allsky/assets/23743816/848d884d-8827-4b02-b6bf-c7e4fca38645)
or do you mean my cameras resolution? 
My camera is 2560x1440

**Reply by Bojack84** [reply]
If you want I can privately send you my external address to help.

**Reply by aaronwmorris** [reply]
I meant what is the resolution of your camera.  If you send me the link, I can probably figure it out.  Thank!

**Reply by Bojack84** [reply]
I sent you the link via your email attached to Github

**Comment by aaronwmorris** [comment, marked as answer]
With the Pi 3, there is a strong possibility that you are running out of memory.  The memory requirements for generating video increase linearly with the output resolution.  You probably have barely enough memory for the 2560x1440 resolution.  You can change the ffmpeg scaling to 75% or 50% and significantly reduce the memory needed to generate the videos.

I would also strongly encourage to have a heatsink and fan to cool the system off.  Generating video uses every CPU and most of the memory on your system, so a lot of heat gets generated and can lock up if it overheats.

**Reply by Bojack84** [reply]
Thank you for the advice.  I am going to set it to 75% and see if that helps.

**Reply by Bojack84** [reply]
As Always Aaron, Thank you for your assistance.  It is working like a charm after setting it to 75%.  One more question: How do I adjust it so the timelapse it creates goes slower with more detail? IE... I want the video to play at like 25% speed so I can catch the airplanes and satellites going overhead?

**Reply by aaronwmorris** [reply]
There are two ways to do this:

- Reduce the FFMPEG frame rate from 25 to something lower.  The lower you make this setting, the more jumpy the video will appear.
- Increase the number of frames by reducing the "exposure period".  This will give the appearance of slowing the video down.  Since you are using an IP camera, the exposure is not really the exposure so this would give you more frames easily.
